packages/grid/x-data-grid/src/tests/columnHeaders.DataGrid.test.tsx

```diff
@@ -49,4 +49,49 @@ describe('<DataGrid /> - Column Headers', () => {
       expect(getColumnHeaderCell(0)).to.have.class('foobar');
     });
   });
+
+  describe('header menu', () => {
```

Comment:
There's already a group for tests related to the column menu in https://github.com/mui-org/material-ui-x/blob/f343bc3c4ca3bd6b314c0609fb778c1f0e5f0956/packages/grid/x-data-grid-pro/src/tests/columnHeaders.DataGridPro.test.tsx#L34

Reply:
I agree with you on that one
If it's available on `DataGrid`, it's better in the `DataGrid` package.